src/main/java/lamball/Storage.java

```diff
@@ -0,0 +1,186 @@
+package lamball;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.List;
+import java.util.Scanner;
+
+
+/**
+ * Contains methods that pertain to file access and modification.
+ *
+ * @author ongzhili
+ */
+public class Storage {
+
+    protected String filePath = "src/main/java/data/list.txt";
+    protected String tempFilePath = "src/main/java/data/tempfile.txt";
+
+    /**
+     * Obtains and initializes list from saved text file locally.
+     *
+     * @param lamb Initialized Lamball chatbot instance.
+     */
+
+    public static void obtainSavedFile(Lamball lamb) {
+        File folder = new File("src/main/java/data");
```

Comment:
Alot of reused path names, can you extract them out and append specific paths to it? i.e. `protected final String ROOT_PATH = "src/main/java/data";`